README.md

```diff
@@ -160,12 +150,4 @@ Now submit the pull request.
 
 Soon I'll be merging all your changes into the master branch of this project. You will get a notification email once the changes have been merged.
 
-## Where to go from here?
-
-Congrats! You just completed the standard _fork -> clone -> edit -> pull request_ workflow that you'll encounter often as a contributor!
-
-Celebrate your contribution and share it with your friends and followers on your social media accounts.
-
-Now let's get you started with your next contribution, wether on this or any other great open source projects on GitHub.
-
 Have fun 👻!
```

Comment:
Please remove this sentence as well.

Reply:
Hi, thanks for the encouragement. Also, could you please specify which sentence you'd like for me to remove?

Reply:
Now I found an easier ways to communicate this issue 😃: 
From the original file, remove everything from **line 25 How to contribute** until the last **line 171 Have fun 👻!**

Reply:
And transfer it to the discussion page right?